pkg/aws/vpc/limits.go

```diff
@@ -2605,36 +2920,6 @@ var Limits = map[string]*VPCLimits{
 		Hypervisor:  "xen",
 		IsBareMetal: false,
 	},
-	"g2.2xlarge": {
```

Comment:
Is this expected to be deleted?

Reply:
% aws ec2 describe-instance-types --instance-types g2.2xlarge

An error occurred (InvalidInstanceType) when calling the DescribeInstanceTypes operation: The following supplied instance types do not exist: [g2.2xlarge]

I think this is deprecated type.

Reply:
Same error for g2.8xlarge as well